README.rst

```diff
@@ -90,7 +89,6 @@ They are kept in separate repos and updated manually by coping:
 * removenbsp
 * `structuredheadings <https://github.com/PolicyStat/ckeditor-plugin-structured-headings>`_
 
-These plugins packed together with CKEditor by symlinking during  update_ckeditor.py run.
```

Comment:
does `update_ckeditor.py` need to be updated to not symlink them now?

Reply:
Yup. `update_ckeditor.py` would only copy now. But I wanted to pass this PR first before pushing changes in PolicyStat